src/main/java/sg/ncl/MainController.java

```diff
@@ -1293,7 +1294,10 @@ public String teams(Model model, HttpSession session) {
 
         TeamManager2 teamManager2 = new TeamManager2();
 
-        List<Image> savedImageList = new ArrayList<>();
+        // stores the list of images created or in progress of creation by teams
+        // e.g. teamNameA : "created" : [imageA, imageB], "inProgress" : [imageC, imageD]
+        Map<String, Map<String, List<Image>>> imageMap = new HashMap<>();
+        boolean isInnerImageMapPresent = false;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "isInnerImageMapPresent". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)